index.js

```diff
@@ -120,6 +120,7 @@ const getDefaults = () => {
     browserstackLaunchers: Object.assign(browserstackLaunchers),
     showQUnitUI: inServerMode(),
     preferHeadless: true,
+    reporters: [ isCI ? 'dots' : 'progress', 'coverage'],
```

Comment:
perhaps we should add 'coverage' after user settings if `coverage` is true? That way the user won't have to worry about manually adding the coverage reporter.

Reply:
Eh, if someone is already making changes they can do the extra step of adding coverage if they need it.